marina/functions/homework.js

```diff
@@ -0,0 +1,70 @@
+/* Our homework was to create functions for arithmetic operations, use them in the file and then console.log results.
+ Every function needs to accept 2 parameters */
+
+let number1;
+let number2;
```

Comment:
You should not declare these variables here, they're not used anywhere, and might lead to some confusion in VSCode.
There is a difference between a variable and function parameter. the:
```
let a;
```
is not used in
```
function (a) {}
```
but instead that `a` from function becomes inner scope of function declaration, and `a` above is..,. well, above it, as a declared function.